tests/fixtures/policy_fixtures.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import json
+from unittest.mock import patch
+
+
+# Sample policy data
+sample_policy_data = {
+    "id": 11,
+    "country_id": "us",
+    "label": None,
+    "api_version": "1.180.1",
+    "policy_json": json.dumps(
+        {"gov.irs.income.bracket.rates.2": {"2024-01-01.2024-12-31": 0.2433}}
+    ),
+    "policy_hash": "NgJhpeuRVnIAwgYWuJsd2fI/N88rIE6Kcj8q4TPD/i4=",
```

Comment:
```suggestion
    "policy_hash": valid_hash_value,
```
__nit:__ This helps keep the data you're testing cleaner